src/actions/executeFetch.js

```diff
@@ -61,6 +64,7 @@ async function getAsyncData({
     defaultOpts || {},
     resourceOpts || {},
     opts || {},
+    { ...!overridableActionMethods.has(actionType) && { method: actionTypeMethodMap[actionType] } },
```

Comment:
My preference would be to encourage the right method but not force it. Not that I don't want to use REST appropriately, but feel like we shouldn't police the customizability. Not a blocker.

Reply:
That's `fetchye`'s role and recommend folks start considering that approach. We should keep the opinions of this library for consistency.